spec/inverted-index-test.js

```diff
@@ -0,0 +1,79 @@
+"use strict";
+
+    //A test suite to read book data
+describe("Inverted Index Suite", function(){
+    let newIndex, books, emptybook, sampleString;
+        
+    beforeEach(function(){
+        //Create an instance of the Index class
+        newIndex = new InvertedIndex();
+        
+        //Test files to be used in the Index
+        books = [
+            {"title": "Heroku",
+            "text":"You will be asked to enter your Heroku credentials the first time you run a command; after the first time, your email address and an API token will be saved"
```

Comment:
**Nitpick**
Line length should not be more than 80 characters. You can use jshint for this